mm2src/coins/hd_wallet/storage/mod.rs

```diff
@@ -344,12 +347,18 @@ mod tests {
         let rick_device0_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "RICK".to_owned(), device0_rmd160)
             .await
             .expect("!HDWalletCoinStorage::new");
+
         let rick_device1_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "RICK".to_owned(), device1_rmd160)
             .await
             .expect("!HDWalletCoinStorage::new");
+
         let morty_device0_db = HDWalletCoinStorage::init_with_rmd160(&ctx, "MORTY".to_owned(), device0_rmd160)
             .await
             .expect("!HDWalletCoinStorage::new");
+        println!(
```

Comment:
Can you use the `debug!` macro instead?

Reply:
deleted..I used for debugging purpose earlier